src/libraries/System.Runtime.Extensions/tests/System/IO/PathTests_Unix.cs

```diff
@@ -21,6 +21,17 @@ public static void GetPathRoot(string value, string expected)
             Assert.Empty(Path.GetPathRoot(value));
         }
 
+        [Fact]
+        public static void GetPathRoot_VerifySameOutput_SpanVsString()
+        {
+            string original = "/home/../folder/../..";
```

Comment:
Instead of adding on-off test for this, I would be better to add the coverage for the Span overload to the tests of the  string overload. It is what we do in other similar cases e.g. in globalization tests.

Reply:
@jkotas need your help. Can you please tell me which tests for the string overload you're talking about or their location?

Reply:
They are in `PathTests_*.cs` in this directory. 

All `Path.GetRoot` tests should look like this: https://github.com/dotnet/runtime/pull/35734/files#diff-d05b8bc6236cdde054bda1d52aa1c063R41 : one line that exercises the string overload and second line that exercises the Span overload.

Reply:
Thanks. Done.